.github/workflows/test-beta.yml

```diff
@@ -0,0 +1,25 @@
+name: test-beta
+
+on:
+  schedule:
+    - cron: "0 6 * * *"
+
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: true
```

Comment:
I think we don't need to limit concurrency since we're restricting this to a daily run.

```suggestion
```